### PR TITLE
Force resolutions for lodash to 4.17.21

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves lodash 14.17.20 security vulnerability reported by Synk

**Overall change:**
- Add ` "lodash": "4.17.21"` to package.json resolutions object

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.100 | [PR#???](https://github.com/bbc/psammead/pull/???) force lodash version to 4.17.21 within package.json resolutions |
```
Comment (RichardPK) [inline CHANGELOG.md]:
```suggestion
| 4.0.100 | [PR#4369](https://github.com/bbc/psammead/pull/4369) force lodash version to 4.17.21 within package.json resolutions |
```

**Review by chris-hinds — APPROVED** [review]

**Comment by andrewscfc** [comment]
I think @jroebu14 was saying this might be fixable without using force resolutions, did you give it a go?

**Comment by RichardPK** [comment]
@andrewscfc I think that was based on the recommendation within synk? 

> Remediation: Your dependencies are out of date, otherwise you would be using a newer lodash than lodash@4.17.20. Try relocking your lockfile or deleting node_modules, reinstalling and running snyk wizard. If the problem persists, one of your dependencies may be bundling outdated modules.

Error path:
```
Introduced through: @bbc/psammead@4.0.99 › @emotion/styled@11.0.0 › @emotion/babel-plugin@11.1.2 › @babel/helper-module-imports@7.12.5 › @babel/types@7.12.12 › lodash@4.17.20
```

We don't have the latest version of all the offending packages,

(Psammead version > latest version)
@emotion/styled 11.0.0 > 11.1.5
@emotion/babel-plugin 11.1.2 > 11.2.0

But I still think we have an issue, as, going down the package hierarchy:

@babel/helper-module-imports latest version is 7.12.13 (updated 21 days ago - lodash 4.7.21 came out 3 days ago)

**Comment by jroebu14** [comment]
I can't remember if I deleted the root `yarn.lock` before running `yarn install`. you could try that

File: CHANGELOG.md (hunk 1 above)
Comment (RichardPK) [inline CHANGELOG.md]:
```suggestion
| 4.0.100 | [PR#4369](https://github.com/bbc/psammead/pull/4369) force lodash version to 4.17.21 within package.json resolutions |
```

**Review by joshcoventry — APPROVED** [review]

**Pull request closed without merging**